_content/monographs/phra-malai-legend-thai-buddhist-literature_brereton-bonnie-pacala.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Phra Malai legend in Thai Buddhist literature: A study of three texts"
+authors:
+  - "Bonnie Pacala Brereton"
+external_url: "https://deepblue.lib.umich.edu/handle/2027.42/128920"
+drive_links:
+  - "https://drive.google.com/file/d/16zOKTreDWyNRGcKqeAXFTDI-LxNwdtYy/view?usp=sharing"
+course: theravada-roots
+tags:
+  - thai-roots
+  - literature
+year: 1992
```

Comment:
monographs require publisher and address. This dissertation should live in booklets and get the `subcat: thesis` field